pkg/manifests/manifests_test.go

```diff
@@ -1433,30 +1433,43 @@ func TestPrometheusQueryLogFileConfig(t *testing.T) {
 			errExpected:      true,
 			volumeExpected:   false,
 		},
-	}
-	for _, tt := range testCases {
+		{
+			name:             "invalid path, relative path",
+			queryLogFilePath: "./dev/query.log",
+			expected:         "",
+			errExpected:      true,
+			volumeExpected:   false,
+		},
+		{
+			name:             "filename only",
+			queryLogFilePath: "query.log",
+			expected:         "query.log",
+			errExpected:      false,
+			volumeExpected:   false,
+		},
+	} {
```

Comment:
You missed to wrap the test with `t.Run()` and I think that it swallows a failing sub-test...

```suggestion
	} {
		t.Run(t.name, func(t *testing.T) {
```